client/my-sites/site-settings/form-general.jsx

```diff
@@ -597,18 +584,19 @@ export class SiteSettingsFormGeneral extends Component {
 								</Button>
 							</div>
 						</CompactCard>
-						{ site && ! isBusiness( site.plan ) && (
-							<Banner
-								feature={ FEATURE_NO_BRANDING }
-								plan={ PLAN_BUSINESS }
-								title={ translate(
-									'Remove the footer credit entirely with WordPress.com Business'
-								) }
-								description={ translate(
-									'Upgrade to remove the footer credit, add Google Analytics and more'
-								) }
-							/>
-						) }
+						{ site &&
+							! isBusiness( site.plan ) && (
```

Comment:
This formatting change suggests that the PR was created before the Prettier 1.15.3 upgrade in #29987 :smile:

Reply:
Fair point 👍 Reformatted in https://github.com/Automattic/wp-calypso/pull/30024/commits/d293582a7bd5803f05ad22948cea4fe108d7a847 and it indeed made the the diff quite smaller.

Reply:
Squashed the commit in the previous one though - no need to complicate the rebase. Will merge when tests pass.